airbyte-integrations/connectors/destination-bigquery-denormalized/Dockerfile

```diff
@@ -17,5 +17,5 @@ ENV ENABLE_SENTRY true
 
 COPY --from=build /airbyte /airbyte
 
-LABEL io.airbyte.version=1.2.20
+LABEL io.airbyte.version=1.3.0
```

Comment:
👍 Following our [versioning guide](https://docs.airbyte.com/contributing-to-airbyte/#semantic-versioning-for-connectors), the addition of a new feature is a minor change